_schema.graphql

```diff
@@ -3303,9 +3312,10 @@ type Fair {
   hours: String
   href: String
   image: Image
+  is_active: Boolean @deprecated(reason: "Prefer isActive instead")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 3315;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_active`

Reply:
cc @alloy — There might be some more diff finessing opportunities for this check. Here a deprecation is triggering the warning. No good deed goes unpunished 😛